### ISSUE TITLE
DAC 28 - Medium Priority WCAG Level AA - Reflow 02

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#28]
<img width="405" alt="DAC_Reflow_02" src="https://user-images.githubusercontent.com/75850106/155569782-42750647-4dac-4a65-a24c-3fe627527936.png">

### Issue
Low vision users are unable to interact with some of the elements that are in the first diagram of ‘Income deprivation in East Staffordshire’ with 400% text resize and 1280px X 1024px browser dimensions. The elements that are positioned off the map lose their interactivity. In some instances, the dots overlap each other, this makes the dot that is overlapped inaccessible.

<img width="599" alt="DAC 28 Current code ref" src="https://user-images.githubusercontent.com/75850106/155571091-638db65a-dfb4-4436-8af6-788e35773043.png">

**Low vision comments:**
 “When looking at the chart with the dots, I realised that this fails reflow, this is because the dots that appear above the dotted lines are not interactive, as they are beyond the container.”

### Suggested Solution
Ensure that the functionality of elements with 400% text resize and 1280px X 1024px browser dimensions is not lost.
